packages/web-components/fast-foundation/docs/guide/webpack.doc.md

```diff
@@ -3,4 +3,217 @@ id: webpack
 title: Webpack
 sidebar_label: Webpack
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-foundation/docs/guide/webpack.doc.md
----
\ No newline at end of file
+---
+
+FAST-DNA works great with TypeScript and Webpack, using a fairly standard setup. Let's take a look at how you can set up a TypeScript+Webpack project, starting from scratch.
+
+## Setting up the Package
+
+First, let's make a directory for our new project. From the terminal:
+
+```shell
+mkdir fast-webpack
+```
+
+Next, let's move into that directory, where we'll set up our project:
+
+```shell
+cd fast-webpack
+```
+
+From here, we'll initialize npm:
+
+```shell
+npm init
+```
+
+Follow the prompts from npm, answering each question in turn. You can always accept the defaults at first and then make changes later in the package.json file.
+
+Next, we'll install the FAST-DNA packages, along with supporting libraries. To do that, run this command:
+
+```shell
+npm install --save @microsoft/fast-components @microsoft/fast-element lodash-es tslib
```

Comment:
I see `@microsoft/fast-components` enumerates `tslib` as a dependency - is this required here?

Reply:
Hmm. Not sure actually. I'll play with removing it. I tried to remove lodash but that was no go.

Reply:
It works without it. However...it is a direct dep of the build when using the `importHelpers` setting in tsconfig...so I'm a bit nervous about removing it, esp. in the case that we change the dependencies of fast-components. If it's ok, I'll leave this one ine.